Sources/Build/BuildPlan.swift

```diff
@@ -587,7 +587,11 @@ public final class SwiftTargetBuildDescription {
 
         extension Foundation.Bundle {
             static var module: Bundle = {
-                return Bundle(path: Bundle.main.bundlePath + "/" + "\(bundleBasename)")!
+                let bundlePath = Bundle.main.bundlePath + "/" + "\(bundleBasename)"
+                guard let bundle = Bundle(path: bundlePath) else {
+                    fatalError("could not load resource bundle: \\(bundlePath)")
```

Comment:
There is a double escape here (`\\`), so this is literally going to print `\(bundlePath)` rather than the bundle path itself.

Reply:
That's actually intentional... this is code that is being emitted into the source file that is generated.  I could switch to use a + if that would make it clearer.

Reply:
(the context here isn't large enough to show it by default, but this is actually all inside a multi-line string literal. The first () is evaluated into the emitted source code, but the second one is evaluated at runtime in the code compiled from the emitted source code).

Reply:
Ah, I misread this, sorry!

Reply:
No problem; thanks for noticing what certainly looked like a mistake.  Without the context of the full source code, and especially without syntax coloring of the multi-line comment, it certainly looked odd.